Makefile

```diff
@@ -149,6 +149,10 @@ core-test: fmt vet manifests
 manager: fmt vet lint manifests
 	go build -o bin/manager ./cmd/core/main.go
 
+# Build server binary
+server: fmt vet lint manifests
+	go build -o bin/vela-server ./pkg/controller/core.oam.dev/v1alpha2/application/testcmd
```

Comment:
why don't we use `make core-run` for test? The application should also be part of vela-core